### DISCUSSION TITLE
Error on running Spring boot in newer minio version. "o.minio.errors.ErrorResponseException: You must specify at least one part"

### DISCUSSION THREAD

**Discussion by ainsooalgown19** [minio/minio#17625] [Q&A] [answered]
Help im encountering error when using spring boot credentials is correct. but giving me this error

Caused by[m: io.minio.errors.ErrorResponseException: You must specify at least one part
    [1mat[m io.minio.MinioClient.execute ([1mMinioClient.java:576[m)
    [1mat[m io.minio.MinioClient.executePost ([1mMinioClient.java:673[m)
    [1mat[m io.minio.MinioClient.completeMultipart ([1mMinioClient.java:1971[m)
    [1mat[m io.minio.MinioClient.putObject ([1mMinioClient.java:1709[m)
    [1mat[m io.minio.MinioClient.putObject ([1mMinioClient.java:1627[m)
    [1mat[m com.beacon.corelate.commons.file.repository.impl.MinioFileRepositoryImpl.putObject ([1mMinioFileRepositoryImpl.java:59[m)
    [1mat[m com

**Comment by klauspost** [comment, marked as answer]
Please open an issue with your calling code to reproduce on [minio-java](https://github.com/minio/minio-java/issues).

This error is returned when no XML or no parts are sent to CompleteMultipartUpload.
